### DISCUSSION TITLE
auth.css served via http

### DISCUSSION THREAD

**Discussion by emsi** [fief-dev/fief#179] [Q&A] [answered]
I've setup self-hosted fief using provided `docker-compose.yaml`. My reverse proxy is providing SSL with Letsencrypt so I'm accessing fief via https (frankly it's odd that the production deployment documentation does does http by default).

When I try to load the login page the auth.css is retrieved via http rather than https which causes the page to fail.

![image](https://user-images.githubusercontent.com/433383/235375138-dd42e530-8f02-4e3a-a860-e9850381122b.png)
 

## Configuration

![image](https://user-images.githubusercontent.com/433383/235375195-44e85a51-9667-4503-8c36-302f40746db6.png)

**Comment by frankie567** [comment, marked as answer]
Hi @emsi 👋 

Since you're behind a proxy, its IP needs to be allowed on the Fief server so it trusts the `X-Forwarded-` headers (and generate links with the right scheme).

For this you'll need to set the `FORWARDED_ALLOW_IPS` environment variable. If you're sure the Fief server is reachable *only* via the proxy, you can set it to `*`.

**Reply by emsi** [reply]
Thank you. 
It solves the issue, however it could be somehow communicated in the UI or appropriate information added to the "production deployment" documentation as production deployment should not be made over unsecured transport layer.

**Reply by frankie567** [reply]
I've added details about this in the docs: https://docs.fief.dev/self-hosting/deployment/ssl/
